packages/manager/src/features/IAM/Shared/Entities/Entities.tsx

```diff
@@ -0,0 +1,105 @@
+import { Autocomplete, Typography } from '@linode/ui';
+import { styled } from '@mui/material/styles';
+import React from 'react';
+
+import { useAccountResources } from 'src/queries/resources/resources';
+
+import { placeholderMap } from '../utilities';
+
+import type {
+  IamAccessType,
+  IamAccountResource,
+  Resource,
+  ResourceType,
+  ResourceTypePermissions,
+} from '@linode/api-v4';
+
+type Props = {
+  access: IamAccessType;
+  type: ResourceType | ResourceTypePermissions;
+};
```

Comment:
We usually use **interfaces** when possible

```suggestion
interface Props {
  access: IamAccessType;
  type: ResourceType | ResourceTypePermissions;
}
```

Reply:
thanks @bnussman-akamai , done